airflow/providers/microsoft/azure/log/wasb_task_handler.py

```diff
@@ -148,7 +146,7 @@ def _read_remote_logs(self, ti, try_number, metadata=None) -> tuple[list[str], l
             self.log.exception("can't list blobs")
 
         if blob_names:
-            uris = [f"wasb://{self.wasb_container}/{b}" for b in blob_names]
+            uris = [f"https://{self.wasb_container}.blob.core.windows.net/{b}" for b in blob_names]
```

Comment:
Overall, the PR looks great! Just curious about why we're changing the URL here?

Reply:
The Azure blob storage connection hook uses https to upload, download log blobs not wasb. Hence I changed it here to avoid any confusion.

Reply:
Sounds good 🙂